cmd/kpod/pull.go

```diff
@@ -26,6 +29,10 @@ var (
 			Name:  "signature-policy",
 			Usage: "`pathname` of signature policy file (not usually used)",
 		},
+		cli.StringFlag{
+			Name:  "authfile",
+			Usage: "Path of the authentication file. Default is ${XDG_RUNTIME_DIR}/containers/auth.json",
```

Comment:
Docker's config file for authentication lives at:  ~/.docker/config.json, should this be named config.json too?  Also should this directory be ~/.containers?

Reply:
XDG_RUNTIME_DIR puts the content into /run/user/UID which makes it more temporal, IE When you log off or the machine reboots, we don't end up with username and passwords in pretty much clear case sitting in the homedir.  I believe the tool will fail over to trying .docker/config.json, but that should be documented in the man page not in the help.

Reply:
Thanks for the reminder, I'd forgotten that nuance.  I like the reasoning, I'm just concerned that most people will expect the file to live in HOME.  We'll just need to document this nicely in a blog or three reminding them that like eating your greens, this is good for you.